src/Util/Test.php

```diff
@@ -520,6 +521,20 @@ public static function getProvidedData(string $className, string $methodName): ?
                         )
                     );
                 }
+
+                try {
+                    $data[$key] = DataSetTransformer::transform($reflector, $value);
```

Comment:
Cannot access array value on non-array variable $data[$key] of type iterable<mixed, mixed>
```
                    $data[$key] = DataSetTransformer::transform($reflector, $value);
                    ^^^^^^^^^^^
```